SU-HSDO/suhumsci
# SHS-5631: Retire legacy codebase

# READY FOR REVIEW

## Summary
- Uninstall all legacy themes
- Remove related configuration
- Update `hs_layouts` module to remove references to `su_humsci_theme`.
- Remove references to `su_humsci_theme` in the documentation

## Need Review By (Date)
06/28

## Urgency
medium

## Steps to Test
1. Setup `hs_traditional` and `hs_colorful` sites locally
2. Confirm that the installation process is successful
3. In both sites, login and visit the "Appereance" admin page
4. Confirm that none of the following themes is installed:
    - Archaeology 8.x (`archaeology`)
    - France-Stanford 8.x (`francestanford`)
    - Mathematics 8.x (`mathematics`)
    - Stanford HumSci Theme 8.x (`su_humsci_theme`)
    - Stanford HumSci SubTheme 8.x (`su_humsci_subtheme`)
5. Confirm that both sites work as expected


## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
